sdk/durabletask/azure-mgmt-durabletask/generated_tests/test_durable_task_mgmt_schedulers_operations.py

```diff
@@ -0,0 +1,123 @@
+# Code generated by Microsoft (R) Python Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+import pytest
+from azure.mgmt.durabletask import DurableTaskMgmtClient
+
+from devtools_testutils import AzureMgmtRecordedTestCase, RandomNameResourceGroupPreparer, recorded_by_proxy
+
+AZURE_LOCATION = "eastus"
+
+
+@pytest.mark.skip("you may need to update the auto-generated test case before run it")
+class TestDurableTaskMgmtSchedulersOperations(AzureMgmtRecordedTestCase):
+    def setup_method(self, method):
+        self.client = self.create_mgmt_client(DurableTaskMgmtClient)
+
+    @RandomNameResourceGroupPreparer(location=AZURE_LOCATION)
+    @recorded_by_proxy
+    def test_schedulers_get(self, resource_group):
+        response = self.client.schedulers.get(
+            resource_group_name=resource_group.name,
+            scheduler_name="str",
+        )
+
+        # please add some check logic here by yourself
+        # ...
+
+    @RandomNameResourceGroupPreparer(location=AZURE_LOCATION)
+    @recorded_by_proxy
+    def test_schedulers_begin_create_or_update(self, resource_group):
+        response = self.client.schedulers.begin_create_or_update(
+            resource_group_name=resource_group.name,
+            scheduler_name="str",
+            resource={
+                "location": "str",
+                "id": "str",
+                "name": "str",
+                "properties": {
+                    "ipAllowlist": ["str"],
+                    "sku": {"name": "str", "capacity": 0, "redundancyState": "str"},
+                    "endpoint": "str",
+                    "provisioningState": "str",
+                },
+                "systemData": {
+                    "createdAt": "2020-02-20 00:00:00",
+                    "createdBy": "str",
+                    "createdByType": "str",
+                    "lastModifiedAt": "2020-02-20 00:00:00",
+                    "lastModifiedBy": "str",
+                    "lastModifiedByType": "str",
+                },
+                "tags": {"str": "str"},
+                "type": "str",
+            },
+        ).result()  # call '.result()' to poll until service return final result
+
+        # please add some check logic here by yourself
+        # ...
+
+    @RandomNameResourceGroupPreparer(location=AZURE_LOCATION)
+    @recorded_by_proxy
+    def test_schedulers_begin_update(self, resource_group):
+        response = self.client.schedulers.begin_update(
+            resource_group_name=resource_group.name,
+            scheduler_name="str",
+            properties={
+                "location": "str",
+                "id": "str",
+                "name": "str",
+                "properties": {
+                    "ipAllowlist": ["str"],
+                    "sku": {"name": "str", "capacity": 0, "redundancyState": "str"},
```

Comment:
why is `redundancyState` here given that this is a read-only property? It should not be included in requests.

Reply:
`generated_tests` is auto generated by codegen and we have created issue to track it https://github.com/Azure/autorest.python/issues/2978. Since it doesn't not influence runtime behavior, please ignore it for now.